polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -930,6 +901,22 @@ class SYCLAccessorVersionBuilder : public SCFLoopVersionBuilder {
 
 //===----------------------------------------------------------------------===//
 
+/// Return the accessor used by \p op if found, and nullptr otherwise.
+static Optional<AccessorType> getAccessorUsedByOperation(const Operation &op) {
```

Comment:
I'd like the static function to be moved at the top of the file so that (a) it can be used throughout the file if we need to and (b) is easy to locate all the static function(s) declared in this file.

Reply:
It is currently with all other static functions, and it is the first static function, so I think the current position is fine.